src/gui/optionsdialog.cpp

```diff
@@ -81,6 +81,74 @@ namespace
+        case QLocale::English:
+            switch (locale.country()) {
+            case QLocale::Australia: return QString::fromUtf8(C_LOCALE_ENGLISH_AUSTRALIA);
+            case QLocale::UnitedKingdom: return QString::fromUtf8(C_LOCALE_ENGLISH_UNITEDKINGDOM);
+            default: return QString::fromUtf8(C_LOCALE_ENGLISH);
+            }
+        case QLocale::Finnish: return QString::fromUtf8(C_LOCALE_FINNISH);
+        case QLocale::French: return QString::fromUtf8(C_LOCALE_FRENCH);
+        case QLocale::Galician: return QString::fromUtf8(C_LOCALE_GALICIAN);
+        case QLocale::Georgian: return QString::fromUtf8(C_LOCALE_GEORGIAN);
+        case QLocale::German: return QString::fromUtf8(C_LOCALE_GERMAN);
+        case QLocale::Greek: return QString::fromUtf8(C_LOCALE_GREEK);
+        case QLocale::Hebrew: return QString::fromUtf8(C_LOCALE_HEBREW);
+        case QLocale::Hindi: return QString::fromUtf8(C_LOCALE_HINDI);
+        case QLocale::Hungarian: return QString::fromUtf8(C_LOCALE_HUNGARIAN);
+        case QLocale::Icelandic: return QString::fromUtf8(C_LOCALE_ICELANDIC);
+        case QLocale::Indonesian: return QString::fromUtf8(C_LOCALE_INDONESIAN);
+        case QLocale::Italian: return QString::fromUtf8(C_LOCALE_ITALIAN);
+        case QLocale::Japanese: return QString::fromUtf8(C_LOCALE_JAPANESE);
+        case QLocale::Korean: return QString::fromUtf8(C_LOCALE_KOREAN);
+        case QLocale::Latvian: return QString::fromUtf8(C_LOCALE_LATVIAN);
+        case QLocale::Lithuanian: return QString::fromUtf8(C_LOCALE_LITHUANIAN);
+        case QLocale::Malay: return QString::fromUtf8(C_LOCALE_MALAY);
+        case QLocale::Norwegian: return QString::fromUtf8(C_LOCALE_NORWEGIAN);
+        case QLocale::Occitan: return QString::fromUtf8(C_LOCALE_OCCITAN);
+        case QLocale::Polish: return QString::fromUtf8(C_LOCALE_POLISH);
+        case QLocale::Portuguese:
+            if (locale.country() == QLocale::Brazil)
+                return QString::fromUtf8(C_LOCALE_PORTUGUESE_BRAZIL);
+            return QString::fromUtf8(C_LOCALE_PORTUGUESE);
+        case QLocale::Romanian: return QString::fromUtf8(C_LOCALE_ROMANIAN);
+        case QLocale::Russian: return QString::fromUtf8(C_LOCALE_RUSSIAN);
+        case QLocale::Serbian: return QString::fromUtf8(C_LOCALE_SERBIAN);
+        case QLocale::Slovak: return QString::fromUtf8(C_LOCALE_SLOVAK);
+        case QLocale::Slovenian: return QString::fromUtf8(C_LOCALE_SLOVENIAN);
+        case QLocale::Spanish: return QString::fromUtf8(C_LOCALE_SPANISH);
+        case QLocale::Swedish: return QString::fromUtf8(C_LOCALE_SWEDISH);
+        case QLocale::Turkish: return QString::fromUtf8(C_LOCALE_TURKISH);
+        case QLocale::Ukrainian: return QString::fromUtf8(C_LOCALE_UKRAINIAN);
+        case QLocale::Uzbek: return QString::fromUtf8(C_LOCALE_UZBEK);
+        case QLocale::Vietnamese: return QString::fromUtf8(C_LOCALE_VIETNAMESE);
+        default:
+            // Fallback to English
```

Comment:
Maybe "Fallback to language name in English"?

Reply:
Maybe it is better to remove this comment?

Reply:
>Maybe it is better to remove this comment?

Ok.

Reply:
Done.